### PR TITLE
Upload device support file for iOS 16.2

### PR DESCRIPTION
**Pull request by bhadjajay23**

### CODE REVIEW

**Review by jobnte — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ Xcode iPhoneOS DeviceSupport (6.0 - 16.0)
 
 **Most recent releases:**</br>
 
+iOS 16.2 - [Download](https://github.com/filsv/iOSDeviceSupport/raw/master/16.1.zip) </br>
```
Comment (dduardo) [inline README.md:7]: @bhadjajay23 
The item described with the download link is not correct.
Item 16.2 is informing version 16.1 for downloading

Comment (marcosgalaviz-d) [reply README.md:7]: :( im waiting for this :(

Comment (nexinexi) [reply README.md:7]: Has anyone checked it? It doesn't work for me on xcode 13.4.1

**Comment by filsv** [comment]
@bhadjajay23 @jobnte @dduardo @marcosgalaviz-d @nexinexi Thanks everyone, here is [16.2 Support Files](https://github.com/filsv/iOSDeviceSupport/raw/master/16.2.zip).

**Pull request closed without merging**